eth/protocols/snap/sync.go

```diff
@@ -2661,7 +2661,7 @@ func (s *Syncer) OnStorage(peer SyncPeer, id uint64, hashes [][]common.Hash, slo
 	for _, node := range proof {
 		size += common.StorageSize(len(node))
 	}
-	logger := peer.Log().New("reqid", id)
+	logger := peer.Log().With("reqid", id)
```

Comment:
The `log.Logger` interface is 'ours', if you add `New` then these changes can be undone

```golang
// A Logger writes key/value pairs to a Handler
type Logger interface {
	// Handler returns the handler associated with this logger
	Handler() slog.Handler

	// TODO: add WithGroup()?

	// With returns a new Logger that has this logger's attributes plus the given attributes
	With(ctx ...interface{}) Logger

	// New returns a new Logger that has this logger's attributes plus the given attributes. Identical to With
	New(ctx ...interface{}) Logger
```